tests/unit/create/via_global_ref/builtin/testing/path.py

```diff
@@ -47,11 +47,15 @@ class PathMockABC(FakeDataABC, Path):
     """Mocks the behavior of `Path`"""
 
     _flavour = getattr(Path(), "_flavour", None)
-
     if hasattr(_flavour, "altsep"):
         # Allows to pass some tests for Windows via PosixPath.
         _flavour.altsep = _flavour.altsep or "\\"
 
+    # Python 3.13 renamed _flavour to pathmod
+    pathmod = getattr(Path(), "pathmod", None)
+    if hasattr(pathmod, "altsep"):
+        pathmod.altsep = pathmod.altsep or "\\"
+
```

Comment:
I could probably combine the _flavour and pathmod thing into a single statement, but leaving this twice IMHO makes it easier to maintain.

Let me know if I should do a `if sys.version_info, else`  thing around those to ease code removal once you stop supporting Python 3.12.

Reply:
`Path.pathmod` *is* the `os.path` module, so this monkeypatches `altsep` everywhere, not just for this mock. Is that intended?

(`Path._flavour` is also global, so the pre-existing code monkeypatches altsep for `pathlib.Path`.)

Reply:
I understood it that _flavour is also os.path, hence this merely ports this hack to 3.13. Will check.

Reply:
Yes inded. If this kills kittens, it did so on Python 3.12 as well.